Source/Core/Core/TitleDatabase.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <fstream>
+#include <iostream>
+
+#include "Core/TitleDatabase.h"
+
+#include "Common/FileUtil.h"
+#include "Common/MsgHandler.h"
+#include "Common/StringUtil.h"
+#include "Core/ConfigManager.h"
+#include "DiscIO/Enums.h"
+
+namespace Core
+{
+static std::string GetLanguageCode(DiscIO::Language language)
+{
+  switch (language)
+  {
+  case DiscIO::Language::LANGUAGE_JAPANESE:
+    return "ja";
+  case DiscIO::Language::LANGUAGE_ENGLISH:
+    return "en";
+  case DiscIO::Language::LANGUAGE_GERMAN:
+    return "de";
+  case DiscIO::Language::LANGUAGE_FRENCH:
+    return "fr";
+  case DiscIO::Language::LANGUAGE_SPANISH:
+    return "es";
+  case DiscIO::Language::LANGUAGE_ITALIAN:
+    return "it";
+  case DiscIO::Language::LANGUAGE_DUTCH:
+    return "nl";
+  case DiscIO::Language::LANGUAGE_SIMPLIFIED_CHINESE:
+    return "zh_CN";
+  case DiscIO::Language::LANGUAGE_TRADITIONAL_CHINESE:
+    return "zh_TW";
+  case DiscIO::Language::LANGUAGE_KOREAN:
+    return "ko";
+  default:
+    return "en";
+  }
+}
+
+TitleDatabase::TitleDatabase()
+{
+  // Titles that cannot be part of the Wii TDB,
+  // but common enough to justify having entries for them.
+
+  // i18n: "Wii Menu" (or System Menu) refers to the Wii's main menu,
+  // which is (usually) the first thing users see when a Wii console starts.
+  m_title_map.emplace("0000000100000002", GetStringT("Wii Menu"));
+  m_title_map.emplace("OHBC", "The Homebrew Channel");
```

Comment:
I don't know how most users refer to it, but the channel name is localized in the Wii menu:

![0000000100000002-1](https://cloud.githubusercontent.com/assets/6716818/26121851/e5bb1c40-3a75-11e7-84b6-6b3ec8e0f9dc.png)
![0000000100000002-2](https://cloud.githubusercontent.com/assets/6716818/26121856/e8803a64-3a75-11e7-83bb-090ba69a5a13.png)

But either way, maybe we should avoid adding a GetStringT for this, since it uses the interface language rather than the console language, leading to inconsistency. Since wiitdb.txt already has two IDs for the Homebrew Channel, I don't see why it can't have the other IDs too.